lib/Twig/Template.php

```diff
@@ -305,6 +305,33 @@ public function getBlocks()
     }
 
     /**
+     * Returns the template source code.
+     *
+     * @return string|false The template source code or false if it can't be retrieved
```

Comment:
I would rather return ``null`` than ``false``. Using a boolean for a not available value is not right semantically. this is what ``null`` is for.